examples/starter.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Most of us are using linux, so a shell script is okay. It might be better to have a python script for this purpose in order to reduce the number of languages and allow other OS, but that's up to you.

Reply:
Writing a pythonic "starter" was also my plan, but it isn't my top priority and i will focus on the big issues first